File: dev-docs/snippets/collections/get-access.swift
import JsBaoClient

// Get the current user's access info for a collection (groups + members).
func getAccess(client: JsBaoClient, collectionId: String) async throws {
  // #region example
  let access = try await client.collections.getAccess(collectionId: collectionId)
  print(access.groups.count, access.members.count)
  // #endregion example
  _ = access
}

File: dev-docs/snippets/collections/get-access.ts
import type { JsBaoClient } from "js-bao-wss-client";

// Get the current user's access info for a collection (groups + members).
export async function getAccess(client: JsBaoClient, collectionId: string) {
  // #region example
  const access = await client.collections.getAccess(collectionId);
  console.log(access.groups.length, access.members.length);
  // #endregion example
  return access;
}
